py/BUILD.bazel

```diff
@@ -227,6 +235,7 @@ py_wheel(
     visibility = ["//visibility:public"],
     deps = [
         ":selenium-pkg",
+        "//:global-license",
```

Comment:
this writes `LICENSE` to the root directory of the `*.whl` - can anyone guide me on getting it into:

`selenium-${SE_VERSION}-dist-info/LICENSE`?